include/swift/Basic/JSONSerialization.h

```diff
@@ -353,13 +358,19 @@ class Output {
   bool PrettyPrint;
   bool NeedBitValueComma;
   bool EnumerationMatchFound;
+  UserInfoMap UserInfo;
```

Comment:
`UserInterprettedConfiguration Config;` or something similar seems to be better naming. It's good to document that this field is not used in a generic way.

Reply:
I was leaning on the Cocoa paradigm of calling a map of arbitrary values user info where it is well established. Should I still change it? In swiftSyntax I'm also planning to call it user info and I feel on that side the name fits nicely.